Cargo.toml

```diff
@@ -12,9 +12,25 @@ keywords = ["bevy", "gamedev", "parallax", "scrolling", "background"]
 exclude = ["assets/*"]
 
 [dependencies]
-bevy = "0.7.0"
 serde = "1.0.136"
 
+[dependencies.bevy]
+version = "0.8.0"
+default-features = false
+features = [
+    "bevy_render",
+    "bevy_core_pipeline",
+    "bevy_sprite",
+    "bevy_asset",
+    "bevy_winit",
+]
+
 [dev-dependencies]
-bevy-inspector-egui = "0.10.0"
+# Temporarily removed until it updates to Bevy 0.8
+# bevy-inspector-egui = "0.11.0"
```

Comment:
`bevy-inspector-egui = "0.12.0"`
I think its ready to go

Reply:
Sweet. I'll get to updating this to use it tomorrow.